src/components/Detail.js

```diff
@@ -0,0 +1,44 @@
+/* eslint camelcase: ["error", {properties: "never"}] */
+/* eslint import/no-extraneous-dependencies: ["error", {"devDependencies": true}] */
```

Comment:
- [ ] The linters are there to enforce code quality and should not be disabled. Kindly enable the linters and fix the issues raised.